haskell.el

```diff
@@ -354,8 +354,11 @@ If `haskell-process-load-or-reload-prompt' is nil, accept `default'."
 (defun haskell-mode-after-save-handler ()
   "Function that will be called after buffer's saving."
   (when haskell-tags-on-save
-    (ignore-errors (when (and (boundp 'haskell-session) haskell-session)
-                     (haskell-process-generate-tags))))
+    (ignore-errors
+      (if (and (boundp 'haskell-session)
```

Comment:
Well, this was my not idea :D
But I think it's not so bad at the end, otherwise user have to generate tags manually.  Do you think we should remove tags generation from jump-to-tag?  _(I suppose there are more surprising things awaiting :D)_